core/dbt/parser/manifest.py

```diff
@@ -622,6 +622,12 @@ def _process_docs_for_macro(
         arg.description = get_rendered(arg.description, context)
 
 
+def _process_docs_for_exposure(
+    context: Dict[str, Any], exposure: ParsedExposure
+) -> None:
+    exposure.description = get_rendered(exposure.description, context)
```

Comment:
mypy raises this error:
```
core/dbt/parser/manifest.py:628: error: Argument 1 to "get_rendered" has incompatible type "Optional[str]"; expected "str"
```

I think the issue is that exposures have optional descriptions that default to `None`, instead of descriptions that default to `''`:

https://github.com/fishtown-analytics/dbt/blob/fec0e31a25d5b922cb1833cffcb5095eb4ee642b/core/dbt/contracts/graph/unparsed.py#L420

Could you try moving that line right below `name` and switching it to:
```
    description: str = ''
```
We may need to update some more unit tests accordingly, since the default values will now differ.